packages/victory-core/src/victory-util/add-events.tsx

```diff
@@ -62,6 +62,7 @@ export interface EventsMixinClass<TProps> {
   ): React.ReactElement;
   cacheValues<TThis>(this: TThis, obj: Partial<TThis>): void;
   getEventState: typeof Events.getEventState;
+  renderData(props: TProps);
```

Comment:
This line wasn't already added as part of previous PRs and was required to get this working.